Makefile

```diff
@@ -3,7 +3,7 @@ PROGNAME=dump1090
 DUMP1090_VERSION ?= unknown
 
 CFLAGS ?= -O3 -g
-DUMP1090_CFLAGS := -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W
+DUMP1090_CFLAGS := -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -Wno-implicit-function-declaration
```

Comment:
What's the failure that requires this?

Reply:
error: implicit declaration of function 'mkstemp' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

Reply:
Why not fix the necessary #include instead?

Reply:
This would be ideal.

Locally I fixed the above error
`#include <stdlib.h> `and `int mkstemp(char *template);`
Now I got another error of the same type with `gettimeofday `

Reply:
I have already solved it.